app/controllers/api/account.php

```diff
@@ -760,6 +760,7 @@
             ->setBody($body)
             ->setFrom($from)
             ->setRecipient($user->getAttribute('email'))
+            ->setSmtp($project->getAttribute('smtp', []))
```

Comment:
Let's use primitives instead of relying on complex structures that are hard to document.

Reply:
updated